compiler/damlc/lib/DA/Cli/Damlc/Command/Damldoc.hs

```diff
@@ -48,6 +49,13 @@ documentation x = Damldoc x <$>
                 <> long "output"
                 <> short 'o'
 
+    optMbPackageName :: Parser (Maybe String)
+    optMbPackageName =
+        option (Just <$> str) $ metavar "NAME"
```

Comment:
```
```suggestion
        optional packageNameOpt
```

Reply:
`packageNameOpt` is positional, where this is a `--package-name` option.

Reply:
Oh good point, still `optional` might make this a bit nicer :slightly_smiling_face:

Reply:
Agreed. Here and in the other flags :-)